pkg/authorization/authorizer/scope/converter_test.go

```diff
@@ -0,0 +1,211 @@
+package scope
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+func TestSpecialEvaluator(t *testing.T) {
```

Comment:
I'm more worried about not having these parts covered with tests and if somebody will try to change one might break it accidentally. But, I won't block the PR for that.